cdap-ui/app/cdap/components/Cloud/Profiles/ListView/ListView.scss

```diff
@@ -70,7 +70,7 @@ $disabled-color: $red-02;
     }
     .grid-body {
       .grid-row {
-        height: $row-height;
```

Comment:
nit: if we're no longer using this variable, then let's remove the variable definition at the top